src/value-equal.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'value-equal' {
```

Comment:
Please move this file under src/@types. I think then TYpeScript will find it automatically and you will not need /// <reference> in router-store.js.